src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -334,6 +338,7 @@ class SpanDetail extends React.Component<Props, State> {
               </Row>
               <Row title="Parent Span ID">{span.parent_span_id || ''}</Row>
               <Row title="Description">{span?.description ?? ''}</Row>
+              <Row title="Status">{span.status || ''}</Row>
```

Comment:
@HazAT I put this below `description`; let me know if another placement is preferred.

Reply:
Nope, that makes sense I think 👍

Reply:
What type is `status`?